solr/prometheus-exporter/build.gradle

```diff
@@ -22,6 +22,7 @@ description = 'Prometheus exporter for exposing metrics from Solr using Metrics
 
 dependencies {
   implementation project(':solr:solrj')
+  runtimeOnly project(':solr:solrj-zookeeper')
   // ideally remove ZK dep
   implementation('org.apache.zookeeper:zookeeper', {
```

Comment:
@HoustonPutman The changes say removed the Zookeeper dependency, but looks like it is still here?

Reply:
I would need to change solrj-zookeeper for that. I'll do it in a separate PR/Issue.

Reply:
ahhh fixed the changelog entry